inst/sql/sql_server/TimeToEventCombined.sql

```diff
@@ -5,7 +5,7 @@ SELECT t.subject_id,
 FROM @cohort_database_schema.@cohort_table t
 LEFT JOIN (
 		SELECT subject_id, min(cohort_start_date) AS cohort_start_date
-		FROM pioneer_sp_stg
+		FROM @cohort_database_schema.@cohort_table
```

Comment:
The hardcoded `pioneer_sp_stg` was referring to the staging table, so shouldn't the parameterised also refer to the staging table? Like this:

```suggestion
		FROM @cohort_database_schema.@cohort_staging_table
```